vassal-app/src/main/java/VASSAL/configure/ConfigureTree.java

```diff
@@ -626,8 +628,8 @@ protected boolean hasChild(Configurable parent, Class<?> childClass) {
   }
 
   protected Action buildAddAction(final Configurable target, final Class<? extends Buildable> newConfig) {
-    AbstractAction action = new AbstractAction("Add " + getConfigureName(newConfig)) {
-      private static final long serialVersionUID = 1L;
+    return new AbstractAction("Add " + getConfigureName(newConfig)) {
+      private static final long serialVersionUID1 = 1L;
```

Comment:
What's the deal here?

Reply:
Hunh. I accepted IntelliJ's offer to "inline" the operation - didn't see it decided to change the little serial version thing. I've put that back.